sdk/python/pulumi_kubernetes/tables.py

```diff
@@ -524,6 +75,7 @@
     "container_port": "containerPort",
     "container_runtime_version": "containerRuntimeVersion",
     "container_statuses": "containerStatuses",
+    "continue_": "continue",
```

Comment:
Any idea why single word IDs are being added here? Seems like those wouldn't be relevant for snake casing.

Reply:
We add a trailing underscore to protected python keywords

https://github.com/pulumi/pulumi/blob/fa3000801ecd25a784029d7e59715a86e441719a/pkg/codegen/python/python.go#L219-L226

Reply:
I'm not sure, to be quite honest. This may be a bug in the casing table calculations.

Reply:
Fix: https://github.com/pulumi/pulumi/pull/4895